model/model.mustache

```diff
@@ -73,25 +74,30 @@ module.exports = {
     if (!{{name}}.rev) {
       return Promise.reject(new httpError('BAD_REQUEST', 'No rev provided'));
     }
-    const valid = schema.validate({{name}});
-    if (!valid) throw new httpError('UNPROCESSABLE_ENTITY');
-    return r.table('{{pluralName}}')
-    .get({{name}}.id)
-    .update(
-      rethinkdb.ifRevMatches({{name}}),
-      { returnChanges: true }
-    )
-    .run()
-    .then(rethinkdb.checkRevError)
-    .then(rethinkdb.firstChange);
+
+    return r.table('{{pluralName}}').get({{name}}.id).run()
+    .then(current => {
+      const updated = Object.assign({}, current, {{name}});
```

Comment:
I don't think this is what we want here. It's actually a problem that we're using `.update` in our reql rather than `.replace`. Ideally we want to be transferring whole documents across the API rather than just the diffs and updating them in.